src/App.css

```diff
@@ -99,3 +99,9 @@ input {
 .row.submit {
   justify-content: center;
 }
+
+#footer
```

Comment:
What is the purpose of the footer? Is it just to show you can find it in a test?